stoobly_agent/test/app/cli/request/response_test.py

```diff
@@ -0,0 +1,45 @@
+import json
+import pdb
+import pytest
+
+from click.testing import CliRunner
+
+from stoobly_agent.test.test_helper import DETERMINISTIC_GET_REQUEST_URL, reset
+
+from stoobly_agent.cli import record, request
+from stoobly_agent.lib.orm.request import Request
+
+@pytest.fixture(scope='module')
+def runner():
+  return CliRunner()
+
+@pytest.fixture(autouse=True, scope='module')
+def settings():
+  return reset()
+
+class TestResponseGet():
+
+  @pytest.fixture(autouse=True, scope='class')
+  def request_create_cli_result(self, runner: CliRunner):
+    record_result = runner.invoke(record, [DETERMINISTIC_GET_REQUEST_URL])
+    assert record_result.exit_code == 0
+    return record_result
+
+  def test_it_returns_correct_response(self, runner: CliRunner):
+    _request = Request.last()
+
+    request_result = runner.invoke(request, ['response', 'get', _request.key()])
+    assert request_result.exit_code == 0
+
+    output = request_result.stdout
+    assert type(output) is str
+    assert type(output) is not bytes
```

Comment:
Do we need both asserts for both `type` and `isinstance`?

Reply:
I think we need at least `type`. `isinstance` supports checking subclasses too which isn't relevant here